authors/apps/profiles/models.py

```diff
@@ -53,6 +56,13 @@ def followers(self):
         profiles = self.to_profile.all()
         return [profile.from_profile.user for profile in profiles]
 
+    @property
+    def highlights(self):
+        highlights = Highlights.objects.filter(profile=self)
+        if len(highlights) < 1:
```

Comment:
The if statement here is not necessary as dumping an empty QuerySet into the serialiser will still return an empty list

Reply:
Removed lines of unnecessary code from the highlights property.